Gemfile

```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
 
-gem "html-proofer"
+gem "html-proofer",
+  # Temporary fork to allow concurrency to be set via CLI.
+  # See: https://github.com/gjtorikian/html-proofer/pull/632
+  github: "patcon/html-proofer",
```

Comment:
I feel a little cautious about this approach

Reply:
ok, def remove it then. i've shared my logic, and your call 🤷

Reply:
wait, no, i thought this was something else.

Sorry, why feeling cautious? This is the self-documenting way I understand folks use package managers when waiting for upstream changes to get merged, and i'm not sure how else one goes about it... waiting on upstream to merge months later is not an approach I'm familiar with. Would forking to hyphacoop org feel more solid?

Reply:
If you want to hold off a bit longer, it will prob be merged in the next week: https://github.com/gjtorikian/html-proofer/pull/632#issuecomment-810670733

But merging stuff like this from in-process forks is totally par for the course in past positions I've worked at, and isn't usually considered a "code smell". It's just house-keeping to go through the lockfile every few months or so and sweep back to mainline or poke PR's along :)